packages/types/src/index.ts

```diff
@@ -481,10 +482,15 @@ export interface EventHistoryCommand {
 }
 export type HTTPMethod = _Method;
 
+export type PrefixCallback = () => string;
+
+export type Prefix = PrefixCallback|string;
```

Comment:
If we _do_ keep `PrefixCallback`, I think it's fine to shorten this to just

```suggestion
export type Prefix = string | () => string;
```

and

Reply:
removed prefixcallback